pkg/api/index.go

```diff
@@ -198,6 +198,16 @@ func (hs *HTTPServer) getNavTree(c *models.ReqContext, hasEditPerm bool) ([]*dto
 		Children:   dashboardChildLinks,
 	})
 
+	navTree = append(navTree, &dtos.NavLink{
```

Comment:
feature flag?

Reply:
done, now using feature flag that enables discussions in general (migrations), and separate flags for each org, dashboard, annotation discussions.

Reply:
removed